FusionMCPBridge/tests/test_core_router.py

```
@@ -278,6 +278,96 @@ class TestGlobalRequestRouter:
+            module_name="test_module"
+        )
+        
+        # Body data should take precedence
+        response = self.router.route_request("/test?key=query_value", "POST", {"key": "body_value"})
+        
+        assert response["status"] == 200
+        assert response["data"]["key"] == "body_value"
+    
+    def test_route_with_path_params_and_query_params(self):
+        """Test routing with both path parameters and query parameters."""
+        def handler_with_both(path, method, data):
```

Ruff (0.14.10)

Unused function argument: `path`

(ARG001)